src/node_platform.h

```diff
@@ -39,7 +39,8 @@ class NodePlatform : public v8::Platform {
   virtual ~NodePlatform() {}
 
   void DrainBackgroundTasks();
-  void FlushForegroundTasksInternal();
+  // returns true iff work was dispatched or executed
```

Comment:
Can you capitalize and punctuate the comment?

Reply:
Sure, done!